src/components/atoms/toast.tsx

```diff
@@ -1,16 +1,12 @@
-import React, { ReactNode } from 'react';
-import {
-  StyleSheet,
-  View,
-  ViewStyle,
-  Text,
-  TextStyle,
-} from 'react-native';
+import React, {ReactNode} from 'react';
+import {StyleSheet, View, ViewStyle, Text, TextStyle} from 'react-native';
 
 import {Markdown} from './markdown';
 
 import {text, colors} from 'theme';
 
+export type ToastType = 'default' | 'success' | 'warning';
```

Comment:
The toast component feels like it needs a refactor to support just success and warning implementations (we don't have any other type of toasts in the app)